data/hai/hai reveal 1 intro.txt

```diff
@@ -73,28 +73,36 @@ mission "Event: Pug Media Coverage"
 					goto nobody
 				`	"I think you both need to shave."`
 					goto scowl
-			`	The right twin rolls his eyes while the left twitches his mustache awkwardly and responds. "Look at this bozo! Bounty's Councilor Maduena Kabiru keeps ranting about defending against unknown dangers at the fringes of human space. Non-stop! Don't we have enough immediate problems without imagining new horrors?"`
+			`	One twin rolls his eyes while the other twitches his mustache awkwardly. He rewinds the video to the Parliamentary discussion, where Danforth is reassuring councilors about the Navy's ability to patrol the fringes of human space.`
+			`	The left twin says, "Look at this bozo! Bounty's Councilor Maduena Kabiru keeps ranting against Admiral uh..." He snaps his fingers trying to remember the name.`
+			`	"Admiral Danforth," says the right twin.`
+			`	"Right. Danforth. This Councilor wants Danforth to defend against unknown dangers at the fringes of human space. Don't we have enough immediate problems without imagining new horrors?"`
 			`	"Like terrorists nuking us," says the right twin.`
 			choice
+				`	"The Navy is skilled and honorable. I trust them to protect us."`
+					goto eyes
 				`	"No, don't worry about that. I ended the bomb threat."`
-				`	"There's a lot of dangers out there, but don't worry. I'll be there to fight them off."`
 				`	"Maybe you're right."`
 					goto shrug
 				`	"All we can do is get on with our lives, right?"`
 					goto shrug
 				`	"If you don't shave soon, people will think you're wearing masks."`
 					goto scowl
 			label nobody
-			`	The right twin says, "Who are you, Admiral Danforth?" The left one laughs and adds, "Did the Navy fire you or are you here on holiday?" Both of them laugh for a few seconds before you can respond.`
+			`	The left twin says, "Who are you, Admiral Danforts?" His twin corrects him. "Danforth. Admiral Danforth. Did the Navy fire you or are you here on holiday?"`
+			`	Both of them laugh for a few seconds before you can respond.`
```

Comment:
```suggestion
			`	Both of them laugh maniacally for a few seconds.`
			`	The left twin says, "Who are you, Admiral Danforts?" His twin corrects him. "Danforth. Admiral Danforth. Did the Navy fire you or are you here on holiday?"`
```
I think starting with their laughter is better. But this might not be the best way to say it.

Reply:
I changed it so they laugh first, but not maniacally.

Reply:
'Maniacally' really gets the point across that they don't believe you at all, or that you're trying to swindle them. It's harder to tell if they just 'laugh'.

Reply:
I'm trying to leave the twins' reactions slightly ambiguous so the player can maintain their own imagined horror of the "uncanny valley" twins.

However, I did just change the description.